src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -4566,4 +4566,65 @@ public LazyWithStatement(Lazy<IOperation> body, Lazy<IOperation> value, bool isI
         public override IOperation Value => _lazyValue.Value;
     }
 
+    /// <summary>
+    /// Represents a local function statement.
+    /// </summary>
+    internal abstract partial class BaseLocalFunctionStatement : Operation, ILocalFunctionStatement
+    {
+        protected BaseLocalFunctionStatement(IMethodSymbol localFunctionSymbol, bool isInvalid, SyntaxNode syntax, ITypeSymbol type, Optional<object> constantValue) :
```

Comment:
Added a note to https://github.com/dotnet/roslyn/issues/19720#issue-230775041 to address this review comment.